src/reporting/clients/IMappingsClient.ts

```diff
@@ -0,0 +1,502 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AccessToken } from "@itwin/core-bentley";
+import { CalculatedProperty, CalculatedPropertyCreate, CalculatedPropertyUpdate } from "../interfaces/CalculatedProperties";
+import { CustomCalculation, CustomCalculationCreate, CustomCalculationUpdate } from "../interfaces/CustomCalculations";
+import { GroupProperty, GroupPropertyCreate, GroupPropertyUpdate } from "../interfaces/GroupProperties";
+import { Group, GroupCreate, GroupUpdate } from "../interfaces/Groups";
+import { Mapping, MappingCreate, MappingUpdate, MappingCopy } from "../interfaces/Mappings";
+import { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IMappingsClient {
+  /**
+   * Gets all Mappings for an iModel.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @param {number} top the number of entities to pre-load.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-mappings/
+   */
+  getMappings(
+    accessToken: AccessToken, 
+    iModelId: string,
+    top?: number
+  ): Promise<Mapping[]>,
+  /**
+   * Gets an async paged iterator of Mappings for an iModel.
```

Comment:
Add sentence: `This method returns an iterator which loads pages of mappings as it is being iterated over.` Similar applies to all *Iterator methods on all clients.

Reply:
added